### ISSUE TITLE
next-sass seems to not work with Next 7

### ISSUE THREAD

**Issue by JonAbrams** [zeit/next-plugins#272]
I get the following when trying to launch my next server:
```
Error: Chunk.entrypoints: Use Chunks.groupsIterable and filter by instanceof Entrypoint instead
```

I've created a [stripped down app](https://github.com/JonAbrams/next-7-sass-fail) to demonstrate the issue. I'm not a webpack 4 wizard, so have no clue how to fix it 😢. Any ideas on how to fix or workaround would be awesome, I really want to upgrade to next@7!

Just calling `withSass` in `next.config.js` seems to be enough to trigger the error, you don't even need to import a scss file.

Problem does not occur with `next-css`.

**Comment by timneutkens** [comment]
Just had to publish the new versions, moving parts, sorry for the delay 🙏

**Comment by JonAbrams** [comment]
Thanks @timneutkens !

**Comment by agungsb** [comment]
Is this solved? I used `"@zeit/next-sass": "^0.2.0",` and `"next": "7.0.0",` and I'm stuck here.
